src/main/java/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration.java

```diff
@@ -21,26 +21,26 @@
 import hudson.util.FormValidation;
 import hudson.util.ListBoxModel;
 import jenkins.model.Jenkins;
+import net.sf.json.JSONObject;
 import org.apache.commons.lang3.StringUtils;
 import org.apache.log4j.Logger;
 import org.jenkinsci.plugins.plaincredentials.StringCredentials;
 import org.kohsuke.stapler.DataBoundConstructor;
 import org.kohsuke.stapler.DataBoundSetter;
 import org.kohsuke.stapler.QueryParameter;
+import org.kohsuke.stapler.StaplerRequest;
 import org.kohsuke.stapler.verb.POST;
 
+import javax.annotation.Nonnull;
```

Comment:
this is not needed, non-null comes from package level so we only annotate things that can be null. Without annotation they are nonnull!